### PR TITLE
Fix weird redirect behavior

### LINKED ISSUE #229: Redirect is broken on Safari and Firefox
Refreshing does nothing to load the schedule. Top bar never completes. Same on iOS. 

screen shot 2014-10-24 at 3 30 25 pm

---

screen shot 2014-10-24 at 3 20 01 pm

### PR DESCRIPTION
**Pull request by brishin**
It turns out that you're not supposed to use a reactive source aka `Session.set`/`Session.get` in routing code. I was using that before, which was causing the redirect function to fail silently if I used the `Session` line.

Should be able to go back from `/schedule` to `/` now as well as be redirected properly from `/` to `/schedule`

Closes #229

### CODE REVIEW

File: client/views/home.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<template name="home">
+If you're looking for your schedule, <a href="/schedule">click here</a>.
```
Comment (schlosser) [inline client/views/home.html:2]: Will a lot of people see this?  It would be nicer if this just redirected them.

**Comment by natebrennand** [comment]
deferring to Dan on this since he knows Meteor better than I

**Comment by schlosser** [comment]
ping @brishin I'm :+1: after you explain a little about when that message is shown

**Comment by brishin** [comment]
So the only time people will see the home template is when they go back from `/schedule` to `/`. 
(Well also for a short second when the app is loading before the initial redirect happens).

Remember we had the problem that Courses was trapping people because it would instantaneously redirect?

**Comment by schlosser** [comment]
OK, so this will allow people to use the back button to go from /schedule
to /, and then it won't immediately redirect them back to /schedule? That
sounds good to me

**Pull request merged**